src/main/java/ca/on/oicr/gsi/dimsum/controller/RestExceptionHandler.java

```diff
@@ -0,0 +1,38 @@
+package ca.on.oicr.gsi.dimsum.controller;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.server.ResponseStatusException;
+
+@ControllerAdvice(basePackages = "ca.on.oicr.gsi.dimsum.controller.rest")
+public class RestExceptionHandler {
+
+    @ExceptionHandler(ResponseStatusException.class)
+    public ResponseEntity<Map<String, String>> handleResponseStatusException(
+            ResponseStatusException ex) {
+        Map<String, String> error = new HashMap<>();
+        error.put("status", ex.getStatus().toString());
+        error.put("error", ex.getReason());
+
+        return new ResponseEntity<>(error, ex.getStatus());
```

Comment:
REST exceptions should be logged similar to MVC exceptions. Again, ideal to just have another method that handles everything, which both of these handler methods can call